CONTRIBUTION.md

```diff
@@ -0,0 +1,24 @@
+# Installation
+
+To start you need to clone [this git repository](https://github.com/CodersCrew/coderscrew-website).  
+Run `npm install` and `npm run dev` for development environment.
+
+## Requirements
+
+Application requires few tools installed on your system
+
+- Node.js - **12.22.0 or later**
+
+# Development
+
+Each change in this repository should be made by pull request. At least one approval is required to merge. Pull requests are merged through `Squash and Merge` to present consistent merge history.  
+Pull request should contain `Closes #<issue number>` text in description for pull request to be linked with issue, eg. `Closes #9` and be titled accordingly to the issue with number in square brackets before, eg. `[9] Add name's rules for PR to readme`. Make sure the linked issue contains all informations about pull request.  
+After merge process every temporary branch will be deleted.
+
+Commit messages should contain useful informations about what was done. Before committing changes make sure tests passed successfully and you used automatic code formatter [Prettier](https://prettier.io/).
```

Comment:
hmm, I think that husy do that, and when tests and etc. don't work correctly then we can't push our changes but we can  add flag --no-verify  🤯🤯🤯. So maybe add to this file that we shouldn't add flag --no-verify.